eclipse-scout-core/test/form/fields/FormFieldSpec.js

```diff
@@ -393,6 +393,32 @@ describe('FormField', () => {
     expect(formField._isSuppressStatusIcon()).toBe(false);
   });
 
+  it('error status', () => {
```

Comment:
Please follow the existing patterns in this file (the test above is not a good example).
Create a describe block for a property or a function: describe('addErrorStatus')
and then a an it block describing what the expectation is: it('adds the given status to the list of error status
and for the new case: it('adds a status with severity error if the parameters is a string

-> Write complete sentences. Write in third person so describe and it create a sentence -> addErrorStatus adds the given status to the list of error status

Please remove the empty lines before the expectations (but keep the ones after the expectations).